src/cloudai/workloads/ai_dynamo/slurm_command_gen_strategy.py

```diff
@@ -27,10 +27,33 @@ class AIDynamoSlurmCommandGenStrategy(SlurmCommandGenStrategy):
 
     def _container_mounts(self) -> list[str]:
         td = cast(AIDynamoTestDefinition, self.test_run.test.test_definition)
+
+        dynamo_repo_path = td.dynamo_repo.installed_path
+        if dynamo_repo_path is None:
+            raise ValueError("dynamo_repo_path is not set - repo may not be installed")
+        dynamo_repo_path = dynamo_repo_path.absolute()
+
         mounts = [
-            f"{td.huggingface_home_host_path}:{td.cmd_args.huggingface_home_container_path}",
+            f"{dynamo_repo_path}:{dynamo_repo_path}",
+            f"{td.cmd_args.huggingface_home_host_path}:{td.cmd_args.huggingface_home_container_path}"
+            if td.cmd_args.dynamo.backend == "vllm"
+            else f"{td.cmd_args.huggingface_home_host_path}:{td.cmd_args.huggingface_home_host_path}",
```

Comment:
Why are these different for vllm? I would assume all backends to have similar way of mounting hugging face path. If we don't have /root/.cache/huggingface as the container path, we risk running into weird filesystem/huggingface-cli issues discovered earlier (HF_HOME not respected, and so on).